README.md

```diff
@@ -41,6 +41,12 @@ You can repeat the configuration block with a different `[name]` if you have mul
 See https://pganalyze.com/docs/install for further details.
 
 
+Setting up a Restricted Monitoring User
+---------------------------------------
+See https://pganalyze.com/docs/install/self_managed/01_create_monitoring_user (or the corresponding
+instructions for your platform) for details.
+
+
 Additional Setup
```

Comment:
I wonder if its also worth dropping the "Additional Setup" part of the README? We don't support the report functionality anymore for which this is intended, and it may be confusing when someone comes here looking for helper SQL to copy.

Reply:
Should we drop the reporting code as well to tidy things a bit? We can go to git history if we decide to revisit that functionality.

Reply:
Yeah, we probably should. If you'd like I can go ahead and rip that out (since I authored most of it I can probably find the spots more easily).

Reply:
Sure, but no rush. In any case, I'll leave that to a future PR.

Reply:
Done that way: https://github.com/pganalyze/collector/pull/537 :)